sp800-63-3/sec5_xAL.md

```diff
@@ -110,9 +112,11 @@ Separating IAL and AAL also works for traditional M-04-04 LOA use cases, such as
 
-Another example where the assessed risk could differ if the agency evaluated the entire business process rather than the online transaction requirements, is a digital service that accepts resumes to apply for open job postings. In this use case, the digital service allows any individual to submit a resume on behalf of anyone else, and in subsequent visits to the site, access the resume for various purposes. Since the resume information is provided in later sessions, and is considered personally identifiable information, the agency must select an AAL that requires multi-factor authentication (MFA). In this case, the requirements of [[EO 13681]](#eo13681) apply, driving the AAL to be at least level 2. However, the identity proofing requirements remain unclear. The entire business process of examining a resume and ultimately interviewing a person requires a significant amount of identity proofing. The agency needs a high level of confidence that the job applicant/interviewee is in fact the subject of the resume submitted online if a decision to hire is made. Yet this level of proofing is not required to submit the resume online. Identity proofing is not required to complete the digital transaction successfully. The resume could be submitted by the applicant, but may also be submitted by a surrogate. Identity proofing the submitter would create more risk than required in the online system as excess personal information would be collected, such as from the surrogate, when no such information is needed. Therefore, the most appropriate IAL selection would be 1. There is no need to identity proof the user to successfully complete the online transaction, nor is there any potential negative impact of offering the online transaction if a proofing failure occurred. Yet, there would be significant impact if the entire business process failed to correctly identity proof the person - a job may be offered to a fraudulent applicant - but that risk should not be applied holistically to the online system via a higher IAL.
+However, the identity proofing requirements may not require the same rigor. The entire business process of examining a resume and interviewing a person requires a significant amount of identity proofing. The agency needs a high level of confidence that the job applicant is in fact the subject of the resume submitted online. Yet proofing is not required to submit the resume. Identity proofing is not required to complete the digital transaction successfully. The resume could be submitted by the applicant, but may also be submitted by a surrogate. Identity proofing the submitter would create more risk than required as personal information would be collected for the surrogate before she could submit any resumes. Therefore, the most appropriate IAL selection would be 1. 
```

Comment:
I'm not sure this should be a separate paragraph, since it refers to the same use example.

The middle sentences seem a little choppy and redundant, suggest instead:

> Yet proofing is not required to submit the resume, since it could be submitted by either the applicant or a surrogate such as a recuiter. Requiring identity proofing of the submitter would create more risk than required as personal information would be required regarding the surrogate before they could submit any resumes. Therefore, the most appropriate IAL selection would be 1.